src/Service/ClientService.php

```diff
@@ -58,8 +68,8 @@ public function __construct(
             $liveEndpointUrlPrefix = $this->systemConfigService->get('AdyenPayment.config.liveEndpointUrlPrefix');
 
         } catch (\Exception $e) {
-            die($e->getMessage());
-            //TODO log error
+            $this->loggerService->addAdyenError($e->getMessage());
+            die();
```

Comment:
Better to use exit if we don't want to print anything. Should we exit here btw or just throw an error so shopware can handle it?

```suggestion
            exit;
```